EIPS/eip-ens-initial-registrar.md

```diff
@@ -0,0 +1,244 @@
+5. **Owned:** After the reveal period has finished, the winning bidder must submit a transaction to finalize the auction, which then calls the ENS's `setSubnodeOwner` function, recording the winning bidder's address as the owner of the hash of the name.
+
+The following table outlines important parameters which define the Registrar's auction mechanism.
+
+#### Registrar Parameters
+
+|        Name        |                                            Description                                             |   Value    |
+|--------------------|----------------------------------------------------------------------------------------------------|------------|
+| totalAuctionLength | The full time period from start of auction to end of the reveal period.                            | 5 days     |
+| revealPeriod       | The length of the time period during which bidding is no longer allowed, and bids must be revealed. | 48 hours   |
+| launchLength       | The time period during which all names will become available for auction.                          | 8 weeks    |
+| minPrice           | The minimum amount of ether which must be locked up in exchange for ownership of a name.           | 0.01 ether |
+
+### Deeds
+
+The Initial Registrar contract does not hold a balance itself. All ether sent to the Registrar will be held in a separate `Deed` contracts. A deed contract is first created and funded when a sealed bid is submitted. After an auction is completed and a hash is registered, the deed for the winning bid is held in exchange for ownership of the hash. Non-winning bids are refunded.
+
+A deed for an owned name may be transferred to another account by its owner, thus transferring ownership and control of the name.
+
+After 1 year of registration, the owner of a hash may choose to relinquish ownership and have the value of the deed returned to them.
+
+Deeds for non-winning bids can be closed by various methods, at which time any ether held will either be returned to the bidder, burnt, or sent to someone else as a reward for actions which help the registrar.
+
+The following table outlines what portion of the balance held in a deed contract will be returned upon closure, and to whom. The remaining balance will be burnt.
+
+#### Refund schedule
+
+| Reason for Deed closure | Refund Recipient | Refund Percentage |
+| --- | --- | --- |
+| A valid non-winning bid is revealed. | Bidder | 99.5% |
+| A bid submitted after the auction period is revealed. | Bidder | 99.5% |
+| An otherwise valid bid is revealed on an owned name. <sup>1</sup> | Bidder | %0.5 |
+| An expired sealed bid is cancelled. <sup>2</sup> | Canceler | 0.5% |
+| A registered hash is reported as invalid. <sup>3</sup> | Reporter | 50% |
+| A registered hash is reported as invalid. <sup>3</sup> | Owner | 50% |
+
+##### Notes:
+
+1. This is to prevent an extortion attack on the current highest bidder, by threatening to reveal a new second highest bid. This forces all bids to be revealed.
```

Comment:
Suggestion:  This incentivizes all bids to be revealed in time.  If bids could be revealed late, an extortion attack on the current highest bidder could be made by threatening to reveal a new second highest bid.